blog/index.html

```diff
@@ -0,0 +1,58 @@
+---
+title: JabRef Blog
+---
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>{{ page.title }}</title>
+	<meta name="keywords" content="JabRef Developers Blog, JabRef Blog, Developers Blog">
+	<meta name="description" content="The Developers Blog of JabRef">
+  <link rel="stylesheet" href="../combo.css">
+  <link rel="stylesheet" href="../blog.css">
+  <link href='http://fonts.googleapis.com/css?family=Raleway:400,300,700' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+  {% if site.favicon %}<link rel="shortcut icon" href="{{ site.favicon }}" type="image/x-icon">{% endif %}
+	{% if site.touch_icon %}<link rel="apple-touch-icon" href="{{ site.touch_icon }}">{% endif %}
+</head>
+<body>
+
+  <div style="position: fixed; left: 10px; top: 10px; z-index: 9999">
+      <a href="../"><i class="fa fa-home fa-2x" style="color:white"></i></a>
+  </div>
+
+  <div id="main">
+      
+    <nav><ul>
+        <li class="p-{{ site.blog.last.id }}"><a href="#{{ site.blog.last.id }}">Top</a></li>
+        <li class="p-welcome"><a href="#welcome">JabRef Blog</a></li>
+        <li class="p-{{ site.blog.first.id }}"><a href="#{{ site.blog.first.id }}">Bottom</a></li>
+    </ul></nav>
+    
+
+    {% for page in site.blog reversed%}
+      {% capture id %}{{ page.id | remove:'/' | downcase }}{% endcapture %}
+      <div id="{{id}}" class="section p-{{id}}">
+        <div class="container {{ page.style }}">
+          <p class="editparagraph">
```

Comment:
I would not add this - since viewers should not be directly invited to modify an existing blog entry